ios/browser/api/ads/ads_client_ios.mm

```diff
@@ -163,11 +167,15 @@
   [bridge_ setLocalStatePref:path value:std::move(value)];
 }
 
-std::optional<base::Value> AdsClientIOS::GetLocalStatePref(
-    const std::string& path) {
+bool AdsClientIOS::GetLocalStatePref(const std::string& path) {
```

Comment:
Is this supposed to be changed? I dont see a change in the bridge

Reply:
Fixed, thanks